.gitignore

```diff
@@ -47,11 +47,12 @@ lib-cov
 logs
 node_modules
 npm-debug.log*
+package-lock.json
 pids
 reports
 screenshots
 target
 **/aggregated-translations/*.*
 lib
 props-table
-package-lock.json
\ No newline at end of file
+package-lock.json
```

Comment:
oh, didn't even see it was specified before. Could have sworn it kept coming up in the `git status` will remove. Thanks